### DISCUSSION TITLE
Api getting started guide

### DISCUSSION THREAD

**Discussion by ajmcateer** [JosefPihrt/Roslynator#811] [Q&A]
Is there a getting started guide for the API? I want to write my own analyzer but I don't see an example of how to get started.

**Comment by josefpihrt** [comment]
Hi, what do you mean by "API"? Do you mean Roslyn API in general (not specific to Roslynator)? Or do you mean Roslynator client libraries like Roslynator.Core that provide additional functionality to Roslyn?
